app/models/album.rb

```diff
@@ -12,6 +12,6 @@ class Album < ApplicationRecord
   normalizes :title, with: -> { _1.squish }
 
   scope :search, ->(q) {
-    where(arel_table[:title].lower.matches("%#{q.downcase}%"))
+    q.blank? ? none : where(arel_table[:title].lower.matches("%#{q.downcase}%"))
```

Comment:
Лучше так (надо использовать фишки скоупов):

```suggestion
    where(arel_table[:title].lower.matches("%#{q.downcase}%")) if q.present?
```